tests/test_csc.py

```diff
@@ -111,11 +129,36 @@ async def test_stop_propagate_laser(self):
                 topic=self.remote.evt_detailedState,
                 detailedState=TunableLaser.LaserDetailedState.NONPROPAGATING,
             )
+            with pytest.raises(salobj.AckError):
+                await self.remote.cmd_stopPropagateLaser.set_start(timeout=STD_TIMEOUT)
+            await self.remote.cmd_startPropagateLaser.set_start(timeout=STD_TIMEOUT)
+            await self.remote.cmd_disable.set_start(timeout=STD_TIMEOUT)
+            await self.remote.cmd_standby.set_start(timeout=STD_TIMEOUT)
```

Comment:
Why these last two lines? If you want to check anything like "the laser stops propagating when I go to disabled state" then please add those checks here.

Does anything interesting change when you go from disabled to standby? If so, again, please test for it. (And if not, consider getting rid of that line).